index.js

```diff
@@ -866,6 +866,7 @@ function buildValue (context, location, input) {
 
   if ((type === undefined || type === 'object') && (schema.anyOf || schema.oneOf)) {
     context.validatorSchemasIds.add(location.getSchemaId())
+    code = 'const errors = []\n'
```

Comment:
```suggestion
    code += 'const errors = []\n'
```

Reply:
That doesn't really make any difference since https://github.com/MoLow/fast-json-stringify/blob/35f3539115d54ed495f610814ac95835407c9b7d/index.js#L865

Reply:
In the future we maybe add some more steps before tis `code =` call, and then we need to debug to realize it was not `=` and not `+=`